docs/source/isaura.rst

```diff
@@ -1,5 +1,168 @@
+
+
+.. _Exracting the topology-related information:
+
+Extracting the topology-related information
+:::::::::::::::::::::::::::::::::::::::::::
+An excellent topological discrimination between signal and background (thanks to the usage of a gaseous medium inside the TPC) is one of the fundamental trademarks of the NEXT experiment. In order to achieve that, it is necessary to (1) separate the different tracks that may form the event, (2) find the extremes for each of them, and (3) compute the energy around these extremes, providing the so-called *blobs*.
+
+In order to compute all the tracking information commented above it will be mandatory first to check that every event contain hits with well-defined energy. For instance, events with all hits outside the krypton correction map boundaries will be thrown away, since their energy cannot be corrected and their ``Ec`` variable (*corrected energy*) will be ``NaN``.
+
+Once the previous check is done, the hits are grouped into 3D volume elements (``voxels``) with the objective of studying the connectivity. The size of these voxels is more or less fixed (depending on the ``strict_vox_size`` parameter in the config file), and their energy correspond to the sum of the energy of the hits included in the voxel. Following a Breadth-First Search (BSF) [#]_ algorithm, the voxels sharing side, edge, or corner will be part of the same **track**. The figure below shows the voxelization result of a real NEXT-White data (Run-VI) single-electron candidate of 1.73 MeV. In this case, after grouping the *deconvoluted hits* into [5 mm x 5 mm x 5 mm] voxels, the event was classified as single-track.
+
+.. _Isaura display:
+
+ .. image:: images/isaura/r8250_evt194237_dhits.png
+   :width: 44.8%
+ .. image:: images/isaura/r8250_evt194237_voxels.png
+   :width: 53%
+
+
+Since events have already been divided into voxel-made 3D tracks, the next (and final) step involves the computation of the **blobs**. They are defined as imaginary 3D spheres located around both ends of each track. Their energy is an excellent tool to investigate whether there has been a large and sudden energy deposition in the track extreme (i.e. *Bragg peak*, indicating the stopping point of a charged particle) or not (starting point of its trajectory). Therefore, this is a crucial stage within the reconstruction chain, since that will allow us to separate double-electron (such as the double-beta signal) from single-electron (the majority of backgrounds) tracks.
+
+
+To compute the position of the blobs, we need to find the two extreme voxels of the track, which is done following the BFS algorithm. Then, the energy-weighted averaged position of the hits inside these voxels will correspond to the **blob center**, as illustrated in the figure below.
```

Comment:
Maybe to add on the figure what is 'blob center' and 'hits'?

Reply:
I would say that the figure is clear, but I'm going to add that information in the text anyway.